examples/simple_xo/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "simple_xo"
```

Comment:
Both _ and - are valid here, but we've been preferring '-' based on what other "good" projects chose to do.

Reply:
Should I do the same with the directory name?

Reply:
No, everywhere else _ is appropriate.